docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+| Dates               | dates                | Deposition date             | deposition_date              | TIME                                    |  --  |   --    |                       MUST                        | N/A                                   | Date when a dataset is initially received by the Data Portal. Also known as submissison Date.                                                                                                            |
+|                     |                      | Release date                | release_date                 | TIME                                    |  --  |   --    |                       MUST                        | N/A                                   | Date when a dataset is made available on the Data Portal                                                                                                                                                 |
+|                     |                      | Last modified date          | last_modified_date           | TIME                                    |  --  |   --    |                     OPTIONAL                      | N/A                                   | Date when a released dataset is last modified                                                                                                                                                            |
+| Cross references    | cross_references     | Related database entries    | related_database_entries     | STRING<sup>**[1](#dataset_list)**</sup> |  --  |   --    |                     OPTIONAL                      | N/A                                   | If a CryoET dataset is also deposited into another database, enter the database identifier here (e.g. EMPIAR-11445). Use a comma to separate multiple identifiers                                        |
+|                     |                      | Related publication DOIs    | dataset_publications         | STRING<sup>**[1](#dataset_list)**</sup> |  --  |   --    |                     OPTIONAL                      | DOI                                   | DOIs of publications related to this dataset.                                                                                                                                                            |
+|                     |                      | Related database links      | related_database_links       | STRING<sup>**[1](#dataset_list)**</sup> |  --  |   --    |                     OPTIONAL                      | DOI                                   | Links to related database entries.                                                                                                                                                                       |
```

Comment:
This actually turned out to be difficult in case of the strain ID, as each host would need to be validated. I think it would be better to create a list of allowable database prefixes in the related_database_entries field and rules for how to build URLs to them.